execution/executor-benchmark/src/lib.rs

```diff
@@ -259,6 +259,7 @@ pub fn run_benchmark<V>(
         );
     }
     if pipeline_config.delay_execution_start {
+        start_vm_time = APTOS_EXECUTOR_VM_EXECUTE_BLOCK_SECONDS.get_sample_sum();
```

Comment:
but it looks like `APTOS_EXECUTOR_VM_EXECUTE_BLOCK_SECONDS` the counter is not supposed to change between L244 and L262?